app/views/declaration/summary/references_section.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import forms.declaration.additionaldeclarationtype.AdditionalDeclarationType
+@import models.requests.JourneyRequest
+@import models.viewmodels.HtmlTableRow
+@import views.ViewDates
+
+@(declarationData: ExportsDeclaration)(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
```

Comment:
Similar

Reply:
For every section we will stay with declarationData as argument. We will use implicit one on the `summaryPage`